syntax = "proto3";
package example;

option java_package = "com.example.grpc";
option java_outer_classname = "HelloWorldServiceProto";
option java_multiple_files = true;
//import public "google/protobuf/timestamp.proto";

// The greeting service definition.
service Greeter {
  // Sends a greeting
  rpc SayHello (HelloRequest) returns (HelloReply) {}
}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string date = 1;
  string type = 2;
  string base64url = 3;
}